packages/terra-alert/src/_variables.scss

```diff
@@ -0,0 +1,28 @@
+:root {
+  // Color variants
+  --terra-alert-alert-background-color: #fed1d1;
+  --terra-alert-alert-shadow-color: #e50000;
+  --terra-alert-alert-title-color: #e50000;
+ 
+  --terra-alert-error-background-color: #fed1d1;
+  --terra-alert-error-shadow-color: #e50000;
+  --terra-alert-error-title-color: #e50000;
+
+  --terra-alert-warning-background-color: #fff3ce;
+  --terra-alert-warning-shadow-color: #ffc20a;
+
+  --terra-alert-advisory-background-color: #f4f4f4;
+  --terra-alert-advisory-shadow-color: #9b9fa1;
+
+  --terra-alert-info-background-color: #f4f4f4;
+  --terra-alert-info-shadow-color: #0092e0;
+
+  --terra-alert-success-background-color: #f4f4f4;
+  --terra-alert-success-shadow-color: #78c346;
+
+  --terra-alert-custom-background-color: #f4f4f4;
+  --terra-alert-custom-shadow-color: #9b9fa1;
+  --terra-alert-custom-text-color: #1c1f21;
+
+  --terra-alert-icon-font-size: 1.3333em;
```

Comment:
Style edits to fix icon/text/button alignment:
--terra-alert-icon-font-size: 1.286em;